ui/src/App.vue

```diff
@@ -106,6 +106,8 @@
                     return null;
                 }
 
+                this.pluginsStore.fetchIcons()
```

Comment:
it disapear? strange?

Reply:
But config is accessible before hand and is used to determine if simple user auth has been setup

Reply:
My bad, I just saw the isLoggedIn protection above it.